pkg/chains/formats/format.go

```diff
@@ -33,6 +33,7 @@ const (
 	PayloadTypeInTotoIte6    config.PayloadType = "in-toto"
 	PayloadTypeSlsav1        config.PayloadType = "slsa/v1"
 	PayloadTypeSlsav2        config.PayloadType = "slsa/v2alpha1"
```

Comment:
Unrelated: should this be named `PayloadTypeSlsav2alpha1` ?

Reply:
We could but we might end up deprecating it so I don't want to change it for now. When we upgrade v2alpha2 to v2, we can do that I suppose?